src/MLOps.NET.Azure/Storage/StorageAccountMetaDataStore.cs

```diff
@@ -143,7 +145,10 @@ public IRun GetRun(Guid runId)
 
             var runTable = tableClient.GetTableReference(nameof(Run));
 
-            return runTable.CreateQuery<Run>().FirstOrDefault(x => x.Id == runId);
+            var run = runTable.CreateQuery<Run>().FirstOrDefault(x => x.Id == runId);
+            run.Metrics = GetMetrics(run.Id);
```

Comment:
I saw that you added this. I think we have method on the `LifeCycleCatalog` that puts these all together in case the user wants a run with metrics etc, but currently these internal methods are flat, meaning they don't pull dependencies with them.

Reply:
reverted